<launch>
	<arg name="found_marker_list_topic" default="/found_marker_list"/>
	<arg name="clear_markers_service" default="/clear_found_markers"/>
	<arg name="camera_namespaces"/>
	<arg name="parent_frame" default="odom"/>

	<arg name="frequency" default="10"/>
	<arg name="fiducial_transform" default="fiducial_transforms"/>

	<node pkg="aruco_finder" name="aruco_list_publisher"
		type="aruco_list_publisher.py" output="screen" respawn="false">
		<param name="found_marker_list" value="$(arg found_marker_list_topic)"/>
		<param name="clear_markers_service" value="$(arg clear_markers_service)"/>
		<rosparam param="camera_namespaces" subst_value="True">$(arg camera_namespaces)</rosparam>
		<param name="parent_frame" value="$(arg parent_frame)"/>

		<param name="frequency" value="$(arg frequency)"/>
		<param name="fiducial_transform" value="$(arg fiducial_transform)"/>
	</node>
</launch>
